docs/support-matrix.md

```diff
@@ -0,0 +1,43 @@
+# Supported Storage Providers
+
+Ark supports sending backup data to a variety of object storage providers, as well as volume snapshots using either a plugin for the volume provider, or [restic][4] for filesystem level backups.
+
+## In-tree providers
+
+Ark supports the follow object and volume storage providers in-tree.
+
+| Provider              | Object Storage       | Block Storage                |
+|-----------------------|----------------------|------------------------------|
+| Amazon Web Services   | S3                   | Elastic Block Store          |
+| Google Compute Engine | Google Cloud Storage | Google Compute block devices |
+| Azure                 | Azure Blob Storage   | Azure Compute block devices  |
+
+## Plugins
+
+Plugins provide Ark a mechanism to use object and volume storage from providers not already supported in tree.
+
+| Provider Name | Plugin Site                                                | Object Storage Provided     | Volume Storage Provided | Ark versions supported |
+|---------------|------------------------------------------------------------|-----------------------------|-------------------------|------------------------|
+| Portworx      | https://docs.portworx.com/scheduler/kubernetes/ark.html    | no                          | yes                     | v0.8.0 to v0.9.0       |
```

Comment:
I wonder if it wouldn't be best to have it as a guide to use the code repo for all plugins.

Reply:
What if the plugin's code isn't public?

Reply:
I would link to the documentation. We could optionally link to the code.

Reply:
Ah I didn't think the plugin could be private. Good point.